feature-libs/organization/administration/components/shared/form/form.component.html

```diff
@@ -6,7 +6,7 @@
     cxOrgItemActive
     [subtitle]="subtitle"
   >
-    <button actions class="button primary" [disabled]="disabled$ | async">
+    <button actions class="button primary" [disabled]="form.disabled">
```

Comment:
AFAIR `disabled$` has been created to resolve issue, that submit button was not unfreezed after get response (but form was). It was especially visible with fail (e.g. id already exist). I can't reproduce this problem now.